libid/fractals.cpp

```diff
@@ -728,3 +733,29 @@ int other_julia_fp_per_pixel()
     }
     return 0;
 }
+
```

Comment:
all new fractal types should be implemented in their own source file, so the burning ship routines should be declared in `burning_ship.h` and implemented in `burning_ship.cpp`